src/core/lib/gprpp/ref_counted_ptr.h

```diff
@@ -71,6 +71,12 @@ class RefCountedPtr {
     value_ = value;
   }
 
+  T* release() {
```

Comment:
Please add a comment for this function to indicate that it is a stopgap measure until all code is written in C++; release is not idiomatic for shared_ptr, unlike the other components of this class.

Reply:
Done.